android/src/main/java/com/mattermost/networkclient/NetworkClientPackage.kt

```diff
@@ -7,7 +7,7 @@ import com.facebook.react.uimanager.ViewManager
 
 class NetworkClientPackage : ReactPackage {
     override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
-        return listOf<NativeModule>(GenericClientModule(reactContext))
+        return listOf<NativeModule>(GenericClientModule(reactContext), APIClientModule(reactContext), NetworkConstants(reactContext))
```

Comment:
what is `NetworkConstants`?

Reply:
The native network constants we pass back to the JS; things like `EXPONENTIAL_RETRY`.